code/controllers/subsystem/persistence.dm

```diff
@@ -183,6 +195,8 @@ SUBSYSTEM_DEF(persistence)
 	CollectSecretSatchels()
 	CollectTrophies()
 	CollectRoundtype()
+	if(CONFIG_GET(flag/use_antag_rep))
+		CollectAntagReputation()
```

Comment:
I can't fault you for putting this here, but it definitely should go after every time antags are picked. Unchanging tickets due to server crashes are unacceptable

Reply:
How ever I do like the idea that server crashing during a round keeps the antag's points as like an automatic antag token for crashes thing.

Maybe add points at the 10 minute (round end logout report) and only subtract here at shutdown

Reply:
@MrStonedOne @Cyberboss For now I'll just leave this as is since you two have conflicting desires and I'm not sure which would be better personally. That means that if the server crashes it's effectively as if the round never happened, no one gains or loses tickets.